torch/csrc/distributed/rpc/init.cpp

```diff
@@ -502,22 +481,12 @@ PyObject* rpc_init(PyObject* /* unused */) {
               requests.
           )")
       .def_readwrite(
-          "map_locations",
-          &TensorPipeRpcBackendOptions::mapLocations,
+          "device_maps",
+          &TensorPipeRpcBackendOptions::deviceMaps,
```

Comment:
I may be wrong here, but this returns a copy of the maps, right? This means that any change the user does to the returned value, it will not be reflected in the options. In other words:
```
// This has no effect
options.device_maps["worker1"]["cuda:0"] = "cuda:1"
// This works
dev_maps = options.device_maps
dev_maps["worker1"]["cuda:0"] = "cuda:1"
options.device_maps = dev_maps
```
I'm fine with either option, just wanted to make sure this was the behavior you intended.

Reply:
I tested it, you are right. Which behavior is preferred? I guess we can make it work by returning a `shared_ptr`?